diff --git a/Src/Board/Board.c b/Src/Board/Board.c
index dbbe066..6d8ace1 100644
--- a/Src/Board/Board.c
+++ b/Src/Board/Board.c
@@ -1012,7 +1012,7 @@ void boardSaveState(const char* stateFile, int screenshot)
     machineSaveState(boardMachine);
 
     // Call board dependent save state
-    boardInfo.saveState(stateFile);
+    boardInfo.saveState();
 
     if (screenshot) {
         bitmap = archScreenCapture(SC_SMALL, &size, 1);
diff --git a/Src/Board/Board.h b/Src/Board/Board.h
index 9e6505d..63038d7 100644
--- a/Src/Board/Board.h
+++ b/Src/Board/Board.h
@@ -67,7 +67,7 @@ typedef struct {
     void   (*destroy)();
     void   (*softReset)();
     void   (*loadState)();
-    void   (*saveState)();
+    void   (*saveState)(void);
     int    (*getRefreshRate)();
     UInt8* (*getRamPage)(int);
     void   (*setDataBus)(void*, UInt8, UInt8, int);
